build_projects/shared-build-targets-utils/DependencyVersions.cs

```diff
@@ -7,9 +7,7 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class DependencyVersions
     {
-        public static readonly string CoreCLRVersion = "1.1.0-beta-24509-04";
-        public static readonly string JitVersion = "1.1.0-beta-24509-04";
-        public static readonly string CoreCLRVersion_Latest = "1.1.0-beta-24415-01";
-        public static readonly string JitVersion_Latest = "1.1.0-beta-24415-01";
+        public static readonly string CoreCLRVersion = "1.1.0-preview1-24512-01";
```

Comment:
Master branch should have the "beta" versions that you removed, while 1.1.0 should have the "preview1" version.